src/views/RSVP.js

```diff
@@ -0,0 +1,220 @@
+import React, { Component } from "react";
+import styled from "@emotion/styled";
+
+import { AppContext } from "../AppProvider";
+import Main from "../components/Main";
+import Article from "../components/Article";
+import Heading from "../components/Heading";
+import FormArea from "../components/FormArea";
+
+const StyledRSVPForm = styled.form`
+  position: absolute;
+  top: 14em;
+  left: 3em;
+  width: 85%;
+`;
+
+const StyledFormInput = styled.input`
+  font-size: 1.5rem;
+  background-color: ${props =>
+    props.error ? "rgba(255, 113, 113, 0.1)" : "#fafcfe"};
+  border: 1px solid ${props => (props.error ? "#FF7171" : "#e6e9ec")};
+  border-radius: 0.125em;
+  height: 2.25em;
+  width: 100%;
+  padding: 0 0.5em;
+  display: block;
+  margin-top: 0.67em;
+  outline: none;
+
+  &:focus {
+    border: 1px solid #152540;
+  }
+`;
+
+const StyledFormTextArea = styled.textarea`
+  font-size: 1.5rem;
+  background-color: #fafcfe;
+  border: 1px solid #e6e9ec;
+  border-radius: 0.125em;
+  height: 6.83em;
```

Comment:
Instead of setting a `height` for the text area, set the `rows` attribute of the text area.

*Example*:
 
```jsx
<textarea rows="4" />
```